src/views/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -190,31 +190,35 @@
   </header>
   {#if !collapsed}
     <main>
-      {#if file.diff.type === "plain" && file.diff.hunks.length > 0}
-        <table class="diff">
-          {#each file.diff.hunks as hunk}
-            <tr class="diff-line">
-              <td colspan={2} />
-              <td colspan={6} class="diff-expand-header">
-                {hunk.header}
-              </td>
-            </tr>
-            {#each hunk.lines as line}
-              <tr class="diff-line" data-expanded data-type={lineSign(line)}>
-                <td class="diff-line-number left" data-type={lineSign(line)}>
-                  {lineNumberL(line)}
+      {#if file.diff.type === "plain"}
+        {#if file.diff.hunks.length > 0}
+          <table class="diff">
+            {#each file.diff.hunks as hunk}
+              <tr class="diff-line">
+                <td colspan={2} />
+                <td colspan={6} class="diff-expand-header">
+                  {hunk.header}
                 </td>
-                <td class="diff-line-number right" data-type={lineSign(line)}>
-                  {lineNumberR(line)}
-                </td>
-                <td class="diff-line-type" data-type={line.type}>
-                  {lineSign(line)}
-                </td>
-                <td class="diff-line-content">{line.line}</td>
               </tr>
+              {#each hunk.lines as line}
+                <tr class="diff-line" data-expanded data-type={lineSign(line)}>
+                  <td class="diff-line-number left" data-type={lineSign(line)}>
+                    {lineNumberL(line)}
+                  </td>
+                  <td class="diff-line-number right" data-type={lineSign(line)}>
+                    {lineNumberR(line)}
+                  </td>
+                  <td class="diff-line-type" data-type={line.type}>
+                    {lineSign(line)}
+                  </td>
+                  <td class="diff-line-content">{line.line}</td>
+                </tr>
+              {/each}
             {/each}
-          {/each}
-        </table>
+          </table>
+        {:else}
+          <div class="binary">Empty file</div>
```

Comment:
class name can be changed.

Reply:
I agree, do you want to do this change in this PR?

Reply:
should I rename `binary` to something else which can be used in both places, or copy `binary` and give it another name? what do you suggest?

Reply:
I think I would use something that can be used in both places.
Maybe `placeholder`?

Reply:
[Updated](https://github.com/radicle-dev/radicle-interface/compare/f3a5908e3247e920cc348cbac711785cddee5c6b..7581e66f162c96b47144a19d48ab983f76a2ca38) to `placeholder`.